index.html

```diff
@@ -45,23 +46,22 @@ <h2 class="puzzle__number">1.</h2>
                 которой&nbsp;ОН фигурирует?</p>
             <form class="form">
                 <label class="form__label" for="first-puzzle">Ваш ответ:</label>
-                <input class="form__field" type="number" min="1800" max="2022" id="first-puzzle" name="fisrt-puzzle"
+                <input class="form__field" type="number" min="1800" max="2022" id="first-puzzle" name="first-puzzle"
                     placeholder="Например: 1984">
-                <div class="form__button-container">
-                    <button class="form__button form__button_type_submit" type="submit">Ответить</button>
-                    <button class="form__button form__button_type_hint" type="button">Узнать ответ</button>
-                </div>
             </form>
-            <div class="wrong-answer-board">
+            <div class="form__button-container">
+                <button class="form__button form__button_type_submit" onclick="handleResponse(1, 'first-puzzle')">Ответить</button>
+                <button class="form__button form__button_type_hint" onclick="handleHint(1)">Узнать ответ</button>
+            </div>
+            <div class="wrong-answer-board" id="wrong-answer-board-1">
                 <p class="wrong-answer-board__text">К&nbsp;сожалению, это неправильный ответ. Попробуйте еще раз!
                     Помните, что можно воспользоваться видеоподсказками или нажать на&nbsp;кнопку &laquo;Узнать
                     ответ&raquo;.</p>
             </div>
-            <div class="correct-answer-board">
-                <p class="correct-answer-board__text">Поздравляем! Начало положено, вы&nbsp;разгадали первое
-                    из&nbsp;семи заданий.</p>
+            <div class="correct-answer-board" id="correct-answer-board-1">
+                <!-- HTML here will be added by JS -->
```

Comment:
It might it confusing but we added these pieces of HTML from JS code; we need this as we don't know the order in which users responses to the questions, therefore we need to do that dynamically.